purchase_discount/tests/test_purchase_discount.py

```diff
@@ -22,16 +22,16 @@ def setUpClass(cls):
         currency_rate_model = cls.env["res.currency.rate"]
         # Set the Exchange rate for the currency of the company to 1
         # to avoid issues with rates
-        latest_currency_rate_line = currency_rate_model.search(
+        la_test_currency_rate_line = currency_rate_model.search(
```

Comment:
This variable name has been changed without a reason.

Reply:
done